app/models/schedule.rb

```diff
@@ -11,6 +11,10 @@ def self.times
     [:morning, :afternoon, :evening]
   end
 
+  def self.correct_size
+    days.size * times.size
+  end
```

Comment:
Why not remove the methods completely?

But bonus points for using `.freeze`, I always forget that 👍